okhttp-clients/src/main/java/com/palantir/conjure/java/okhttp/RemotingOkHttpCall.java

```diff
@@ -245,6 +247,13 @@ public void onResponse(Call call, Response response) throws IOException {
                     return;
                 }
 
+                if (isStreamingBody(call)) {
+                    callback.onFailure(
+                            call,
+                            new HttpRetryException("Cannot retry streamed HTTP body", response.code()));
```

Comment:
Question: I think it would be nice to attach some rendering of the body here, so we know what the response actually was. Like the test responds with 503, so host is up: would be nice to know that. Suggestions?

Reply:
this will never run given the check at the top: https://github.com/palantir/conjure-java-runtime/pull/1152/files#diff-4c0c5883336d4eff08c9bc33ba1059c0R188

Reply:
Can we move this to the top and add the exception parameter as a suppressed exception to this one?
Also can we make this a SafeIoException instead, so it's loggable?

Reply:
That is in another method :) That check is in onFailure, this is onResponse. Sometimes reading diff in github bites you

Reply:
bah, tricked!

Reply:
I'll happily make this a SafeIoException tomorrow